content/2_Bronze/Intro_Sets.mdx

```diff
@@ -873,4 +945,224 @@ make their implementation easier.
   </Quiz.Question>
 </Quiz>
 </CPPSection>
+
+<JavaSection>
+<Quiz>
+  <Quiz.Question>
+    What is the time complexity of insertions, deletions, and searches in a sorted set of size $N$?
+    <Quiz.Answer>
+      $\mathcal{O}(1)$
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $\mathcal{O}(N \log N)$
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $\mathcal{O}(N)$
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      $\mathcal{O}(\log N)$
+      <Quiz.Explanation>
+    Correct.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    Which of the following is the correct output of this code?
+  ```java
+  Set<Integer> s = new TreeSet<>();
+  s.insert(1);
+  s.insert(4);
+  s.insert(5);
+  s.insert(2);
+  System.out.println(s.contains(3));
+  ```
+    <Quiz.Answer>
+      `true`
+      <Quiz.Explanation>
+
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      `false`
+      <Quiz.Explanation>
+        Correct. $3$ isn't present in `s`.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      `null`
+      <Quiz.Explanation>
+
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      Error: `NullPointerException`
+      <Quiz.Explanation>
+
+      </Quiz.Explanation>
+    </Quiz.Answer>
+
+  </Quiz.Question>
+  <Quiz.Question>
+    Which of the following is the correct output of this code?
+  ```java
+  Set<Integer> s = new TreeSet<>(Arrays.asList(4, 1, 2, 1, 3, 2, 3, 1, 3, 1));
+  System.out.println(s.size());
+  ```
+    <Quiz.Answer>
+      $1$
+      <Quiz.Explanation>
+
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      $4$
+      <Quiz.Explanation>
+        Correct. There are $4$ unique elements in `s`.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $7$
+      <Quiz.Explanation>
+
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $10$
+      <Quiz.Explanation>
+    Sets only store unique elements.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+
+  </Quiz.Question>
+  <Quiz.Question>
+    What is the time complexity of insertions, deletions, and removals in an unordered map of size $N$?
```

Comment:
aren't removals and deletions the same?? maybe `insertions, deletions, and searches`